DESIGN.md

```diff
@@ -0,0 +1,147 @@
+# Design
+
+Utapi tracks metrics of a service's usage. Metrics provided by Utapi include the
+number of incoming and outgoing bytes, the number of objects being stored, the
+storage utilized in bytes, and a count of operations performed on a service's
+resources. Operations supported by Utapi include APIs offered by Scality's [S3
+Server](https://github.com/scality/S3). Metrics can be retrieved for a given
+time range in a service's history.
+
+## Time Range
+
+Utapi offers metrics for a time range provided by the user. For example, Utapi
+allows a user to view all actions occurring over the course of a particular
```

Comment:
view all actions ~~occurring~~ that have occurred over the course of a particular